service/routes.py

```diff
@@ -7,6 +7,7 @@
 from flask import Flask, jsonify, request, url_for, make_response, abort
 from service.common import status  # HTTP Status Codes
 from service.models import Inventory
```

Comment:
Slight nitpick but can we change this to:
```suggestion
from service.models import Inventory, Condition
```

Reply:
Ok, I can make this change in the next sprint